mac

```diff
@@ -169,8 +168,9 @@ install_asdf_plugin() {
 source "$HOME/.asdf/asdf.sh"
 install_asdf_plugin "ruby" "https://github.com/asdf-vm/asdf-ruby.git"
 install_asdf_plugin "nodejs" "https://github.com/asdf-vm/asdf-nodejs.git"
+install_asdf_plugin "postgres" "https://github.com/smashedtoatoms/asdf-postgres.git"
 
-install_asdf_language() {
+install_asdf_language_latest() {
```

Comment:
While this is more descriptive I think I would prefer to not make this change. Maybe it's already too late, but it would be nice to be able to pull in changes from the original repo more easily.